source/online_usecases/brain_area_circuit_in_silico_experiments/hippocampus/configure_run_brainregion_preconf_model_data/configure_run_brainregion_preconf_model_data.rst

```diff
@@ -6,102 +6,88 @@ Configure and run a rat hippocampus CA1 region using preconfigured HBP model and
 
 This use case allows a user to configure and launch a simulation on HBP HPAC infrastructure using `Unicore API <https://www.unicore.eu/>`__.
 
+===============
 Run Simulations
 ===============
 
 |run_simulation|
 
-----------------
-
-In the **top section** you can configure:
 
-   |run_simulation_top|
+In the **Section 1)** you can configure:
 
 -  ``Population``: Population of cells to be loaded by the simulator.
 -  ``Duration``: Time length of stimulus duration, given in
    milliseconds(ms)
 -  ``View Simulations``: From here you can see all the history of
-   simulations (explained in section **View Simulations**)
+   simulations (explained in page `View Simulations <#view-simulations>`_)
 
---------------
 
-In the **next section** you can configure:
+In the **Section 2)** you can configure:
 
-   |run_simulation_middle|
 
--  ``add`` new stimulus/report, ``edit`` or ``remove`` (You can also
+-  Using the buttons |edit_buttons| you can ``add``, ``edit`` or ``remove`` new stimulus/report (You can also
    double-click on the timeline to add or edit)
 
-   |edit_buttons|
-
-
--  ``Stimulations``
-
-   -  For creation or edition of the item a form will appear
+-  ``Stimulations`` configuration:
 
-      -  *Population*: Name of a population to receive the stimulation
-      -  *Delay*: Time when the stimulus commences, given in
-         milliseconds(ms)
-      -  *Duration*: Time length of stimulus duration, given in
-         milliseconds(ms)
-      -  *Synapses per cell*: The number of synapses to create
-      -  *Pattern*: Type of stimulus to inject
-      -  *Lambda*: Configure the random distribution
-      -  *Weight*: The strength of the created synapses
+  -  *Population*: Name of a population to receive the stimulation
+  -  *Delay*: Time when the stimulus commences, given in
+     milliseconds(ms)
+  -  *Duration*: Time length of stimulus duration, given in
+     milliseconds(ms)
+  -  *Pattern*: Type of stimulus to inject
+  -  *Synapses per cell*: The number of synapses to create
+  -  *Lambda*: Configure the random distribution
+  -  *Weight*: The strength of the created synapses
 
-         |edit_stimulus|
+    |edit_stimulus|
 
 -  ``Reports``
 
-   -  For creation or edition of the item a form will appear
+  -  *Population*: Defines the region from where the data will be reported.
+  -  *StartTime*: Time to start reporting(ms)
+  -  *EndTime*: Time to stop reporting(ms)
+  -  *Compartments*: Compartment means that each compartment outputs
```

Comment:
@alex4200 What do you suggest to put in this one?

Reply:
I would use the terms in quotes :
'compartment': each compartment will report separately  in the report file
'synapse': each synapse will report separately in the report file